<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastrePageReportDialog</class>
 <widget class="QDialog" name="CadastrePageReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>59</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="cpage_twidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>718</width>
     <height>292</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PrintDate</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Page Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Person ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Right Holder</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Parcel ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Streetname-Khashaa</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="results_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="print_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastre page report</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="print_year_chbox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Print</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="print_year_sbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>59</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2000</number>
   </property>
   <property name="maximum">
    <number>2100</number>
   </property>
   <property name="value">
    <number>2017</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Person ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="person_id_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="parcel_id_edit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcel ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
